README.md

```diff
@@ -2,3 +2,163 @@
 
 An initial scalable filesystem for Linux operating systems
```

Comment:
We should rework this section with a quick introduction to what initoverlayfs is.

Reply:
@rhatdan @ericcurtin added a short sentence, please review!